### ISSUE TITLE
Dependency cycle?

### ISSUE THREAD

**Issue by threesquared** [tpadjen/ng2-prism#1]
Trying to install with angular `2.0.0-beta.7` i get `Peer ng2-prism@2.0.2 wants angular2@2.0.0-beta.6` however when using `2.0.0-beta.6` i get `Peer angular2@2.0.0-beta.6 wants reflect-metadata@0.1.2` and `Peer ng2-prism@2.0.2 wants reflect-metadata@^0.1.3`

**Comment by tpadjen** [comment]
Should be fixed by 4c1db77 to work with angular2 beta 7. I published the update to the npm package as version 2.0.3 . Try this one-liner to get the correct versions (the same as those listed in the [quickstart](https://angular.io/docs/ts/latest/quickstart.html) plus ng2-prism@2.0.3):

```
npm i angular2@2.0.0-beta.7 es6-promise@3.0.2 es6-shim@0.33.3 reflect-metadata@0.1.2 rxjs@5.0.0-beta.2 zone.js@0.5.15 ng2-prism@2.0.3
```

Also, it is likely to still work with any recent version of the peer dependencies, as long as angular2's polyfill conditions are met. See [this comment](https://github.com/angular/angular/issues/6659#issuecomment-185120027) for more info.

**Comment by threesquared** [comment]
Amazing, thank you.
